woo-razorpay.php

```diff
@@ -171,7 +171,7 @@ public function __construct($hooks = true)
             $isAccCreationAvailable = false;
 
             // Load preference API call only for administrative interface page.
-            if (is_admin())
+            if (current_user_can('administrator'))
```

Comment:
How this makes a difference? @poojarazor

Reply:
Above condition is not working properly @ChetanGN